packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss

```diff
@@ -18,20 +18,72 @@
     .inline-input-with-button {
       display: flex;
       flex-direction: row;
+      margin: 10px 61px;
+      width: 95%;
 
       .iui-input-container.iui-inline > .iui-input:last-child {
         min-width: 300px;
         padding-right: 61px;
       }
 
       .iui-input-container {
-        grid-template: "label inputs icon" / auto minmax(50px, 290px) auto;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        font-family: "Open Sans",-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Helvetica Neue",Arial,sans-serif;
```

Comment:
wait, do you really need to set all this extra stuff? 
is this diverging too much from the design standards made by the components?

Reply:
Good point, defining the family isn't necessary but explicitly inheriting font-family and font-size is needed to change iui-input and iui-tab.

Reply:
any reason the font family or size needs to be changed at all?

Reply:
Seems to match better

font-family + font-size Inherited: 
![image](https://user-images.githubusercontent.com/55515768/234983968-311dd701-abde-4e31-8c58-0b9dd7c097b1.png)

Without: 
![image](https://user-images.githubusercontent.com/55515768/234983586-f49c6b33-ba45-4ba9-ab20-421161b42c6e.png)

Reply:
ah dont worry too much about the exact look, the link i pointed too is using like a 2 yr old itiwnui ver
i wouldnt try to override the look too much, defeats the purpose of having our own custom library / design system